django_scrubber/management/commands/scrub_validation.py

```diff
@@ -13,8 +14,17 @@ def handle(self, *args, **options):
 
         found_models = 0
         found_fields = 0
+
+        whitelisted_models = settings_with_fallback('SCRUBBER_VALIDATION_WHITELIST')
+
         if len(non_scrubbed_field_list):
             for model_path, affected_field_list in non_scrubbed_field_list.items():
+
+                if model_path in whitelisted_models:
+                    print(f'Model {model_path!r} was excluded via \'SCRUBBER_VALIDATION_WHITELIST\' and will '
+                          f'not be validated.')
```

Comment:
f-string is unnecessary here.

Reply:
The `!r` notation takes care of the ticks, don't you think that's a plus?

Reply:
Sorry, I just meant the second line of the string.
`f'not be validated.'` could be `'not be validated.'`

Reply:
fixed.